mixnet/README.md

```diff
@@ -0,0 +1,113 @@
+# NomMix Simulation
+
+* [Project Structure](#project-structure)
+* [Features](#features)
+* [Future Plans](#future-plans)
+* [Installation](#installation)
+* [Getting Started](#getting-started)
+
+## Project Structure
+
+- `cmd`: CLIs to run the simulation and analyze the results.
+- `sim`: Simulation that runs the NomMix defined in the `protocol` package.
+- `protocol`: Core NomMix protocol implementation, which is going to be moved to the [nomos-repos](https://github.com/logos-co/nomos-specs) repository once verified by simulations.
+- `framework`: Asynchronous framework that provides essential async functions for simulations and tests, implemented with various async libraries ([asyncio](https://docs.python.org/3/library/asyncio.html), [μSim](https://usim.readthedocs.io/en/latest/), etc.)
+
+## Features
+
+- NomMix protocol simulation
+- Performance measurements
+  - Bandwidth usages
+- Privacy property analysis
+  - Message sizes
```

Comment:
Yes. I thought that our design ensures that the message size is always the same to prevent the adversary from knowing what stage the packet was currently in.